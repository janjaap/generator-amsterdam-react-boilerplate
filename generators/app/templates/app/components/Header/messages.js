import { defineMessages } from 'react-intl';
import { scope } from '../../i18n';

export default defineMessages({
  logged_in_as: {
    id: `${scope}.logged_in_as`,
    defaultMessage: 'Logged in as',
  },
  log_in: {
    id: `${scope}.log_in`,
    defaultMessage: 'Log in',
  },
  log_out: {
    id: `${scope}.log_out`,
    defaultMessage: 'Log out',
  },
});
