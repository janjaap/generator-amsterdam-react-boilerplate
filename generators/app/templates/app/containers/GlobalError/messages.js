import { defineMessages } from 'react-intl';
import { scope } from '../../i18n';

export default defineMessages({
  unable_to_fetch: {
    id: `${scope}.unable_to_fetch`,
  },
  unauthorized: {
    id: `${scope}.unauthorized`,
  },
  session_expired: {
    id: `${scope}.session_expired`,
  },
  unknown_error: {
    id: `${scope}.unknown_error`,
  },
  server_error: {
    id: `${scope}.server_error`,
  },
  service_unavailable: {
    id: `${scope}.service_unavailable`,
  },
  resource_not_found: {
    id: `${scope}.resource_not_found`,
  },
  no_vbo_data_available: {
    id: `${scope}.no_vbo_data_available`,
  },
  login_failed: {
    id: `${scope}.login_failed`,
  },
  logout_failed: {
    id: `${scope}.logout_failed`,
  },
  authorize_failed: {
    id: `${scope}.authorize_failed`,
  },
  error_reporting_title: {
    id: `${scope}.error_reporting_title`,
  },
  error_reporting_subtitle: {
    id: `${scope}.error_reporting_subtitle`,
  },
  error_reporting_label_name: {
    id: `${scope}.error_reporting_label_name`,
  },
  error_reporting_label_email: {
    id: `${scope}.error_reporting_label_email`,
  },
  error_reporting_label_comments: {
    id: `${scope}.error_reporting_label_comments`,
  },
  error_reporting_label_close: {
    id: `${scope}.error_reporting_label_close`,
  },
  error_reporting_label_submit: {
    id: `${scope}.error_reporting_label_submit`,
  },
  error_reporting_success_message: {
    id: `${scope}.error_reporting_success_message`,
  },
  error_reporting_error_generic: {
    id: `${scope}.error_reporting_error_generic`,
  },
  error_reporting_error_form: {
    id: `${scope}.error_reporting_error_form`,
  },
});
