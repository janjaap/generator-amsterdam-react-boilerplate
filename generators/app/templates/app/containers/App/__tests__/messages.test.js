import messages from '../messages';

describe('containers/App/messages', () => {
  it('should match the snapshot', () => {
    expect(messages).toMatchSnapshot();
  });
});
