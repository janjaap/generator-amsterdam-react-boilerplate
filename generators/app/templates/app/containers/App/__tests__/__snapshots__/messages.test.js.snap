// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/App/messages should match the snapshot 1`] = `
Object {
  "accommodation_object_id": Object {
    "id": "registraties.accommodation_object.id",
  },
  "address": Object {
    "id": "registraties.address",
  },
  "amount_of": Object {
    "id": "registraties.amount_of",
  },
  "area": Object {
    "id": "registraties.area",
  },
  "bag_objects": Object {
    "id": "registraties.bag_objects",
  },
  "brk_objects": Object {
    "id": "registraties.brk_objects",
  },
  "cadastral_object_indication": Object {
    "id": "registraties.cadastral_object.indication",
  },
  "cadastral_object_indications": Object {
    "id": "registraties.cadastral_object.indications",
  },
  "chamber_of_commerce_nr": Object {
    "id": "registraties.chamber_of_commerce.nr",
  },
  "chamber_of_commerce_nrs": Object {
    "id": "registraties.chamber_of_commerce.nrs",
  },
  "close": Object {
    "id": "registraties.close",
  },
  "csv_file_name": Object {
    "id": "registraties.csv_file_name",
  },
  "export_cta": Object {
    "id": "registraties.export_cta",
  },
  "extra_fields": Object {
    "id": "registraties.extra_fields",
  },
  "filled_in_by": Object {
    "id": "registraties.filled_in_by",
  },
  "house_id": Object {
    "id": "registraties.house.id",
  },
  "no": Object {
    "id": "registraties.yes",
  },
  "note": Object {
    "id": "registraties.note",
  },
  "note_remark": Object {
    "id": "registraties.note_remark",
  },
  "number_identification_id": Object {
    "id": "registraties.number_identification.id",
  },
  "on_this_page": Object {
    "id": "registraties.on_this_page",
  },
  "overview": Object {
    "id": "registraties.overview",
  },
  "public_space": Object {
    "id": "registraties.public_space",
  },
  "public_space_id": Object {
    "id": "registraties.public_space_id",
  },
  "report_feedback": Object {
    "id": "registraties.report_feedback",
  },
  "rsin": Object {
    "id": "registraties.rsin",
  },
  "rsins": Object {
    "id": "registraties.rsins",
  },
  "to_stelselpedia": Object {
    "id": "registraties.to_stelselpedia",
  },
  "unknown": Object {
    "id": "registraties.unknown",
  },
  "yes": Object {
    "id": "registraties.yes",
  },
}
`;
